src/java/org/apache/cassandra/service/StartupChecks.java

```diff
@@ -187,6 +191,66 @@ public void verify(StartupChecksOptions options) throws StartupException
         }
     }
 
+    // https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1057843
+    public static final StartupCheck checkKernelBug1057843 = new StartupCheck()
+    {
+        private final Range<Semver> affectedKernels = Range.closedOpen(
```

Comment:
why can not be these variables just local in the scope of `execute` method? They are private and final, we do not change them anywhere and they are local to the check itself, used just in one place.

Reply:
it is just my sense of aesthetic - those are the constant input data, while the method contains the logic; I can change that if you insist

Reply:
though, changing that will require probably another run of all tests...